tests/test-buffer-from-handle.cpp

```
@@ -3,7 +3,7 @@
 using namespace rhi;
 using namespace rhi::testing;
 
-GPU_TEST_CASE("buffer-from-handle", D3D12 | Vulkan | Metal)
+GPU_TEST_CASE("buffer-from-handle", D3D12 | Vulkan | Metal | CUDA | WGPU)
```

Cppcheck (2.21.0)

[error] There is an unknown macro here somewhere. Configuration is required. If DOCTEST_GLOBAL_NO_WARNINGS is a macro then please configure it.

(unknownMacro)